apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.sql

```diff
@@ -0,0 +1,101 @@
+-- BLOCK select_assessment_access_policies
+SELECT
+  format_date_full_compact (
+    aap.created_at,
+    coalesce(ci.display_timezone, c.display_timezone)
+  ) AS created_at,
+  aap.created_by::text AS created_by,
+  aap.credit::text AS credit,
+  format_date_full_compact (
+    aap.end_date,
+    coalesce(ci.display_timezone, c.display_timezone)
+  ) AS end_date,
+  aap.note AS note,
+  format_date_full_compact (
+    aap.start_date,
+    coalesce(ci.display_timezone, c.display_timezone)
+  ) AS start_date,
+  (
+    SELECT
+      name
+    FROM
+      groups
+    WHERE
+      id = aap.group_id
+  ) AS group_name,
+  (
+    SELECT
+      uid
+    FROM
+      users
+    WHERE
+      user_id = aap.user_id
+  ) AS student_uid,
+  aap.id AS id
+FROM
+  assessment_access_policies AS aap
+  JOIN assessments AS a ON (a.id = aap.assessment_id)
+  JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
+  JOIN pl_courses AS c ON (c.id = ci.course_id)
+WHERE
+  assessment_id = $assessment_id
+ORDER BY
+  aap.created_at;
```

Comment:
Nit: it may make more sense to sort these in `DESC` order instead of the default of `ASC`. My thinking is that if someone creates a new override, they'd want to see it at the top of the page. This is like GitHub issues or pull requests.

Reply:
Fixed in [95551e1](https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/95551e1e93cc1b86404fa0e39edb99bc67224473)